packages/ui/src/Fab/Fab.tsx

```diff
@@ -0,0 +1,10 @@
+import { forwardRef } from 'react';
+
+import { StyledFab } from './styled';
+import { FabProps } from './types';
+
+export const Fab = forwardRef<HTMLButtonElement, FabProps>((props, ref) => {
+  return <StyledFab {...props} ref={ref} />;
+});
+
+export default Fab;
```

Comment:
[style guide](https://astraltrack.atlassian.net/wiki/spaces/FG/pages/3559522553/Import+export#%D0%98%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D1%8E%D1%82%D1%81%D1%8F-%D0%B8%D0%BC%D0%B5%D0%BD%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B5-%D1%8D%D0%BA%D1%81%D0%BF%D0%BE%D1%80%D1%82%D1%8B)